src/notation/view/playbackcursor.cpp

```diff
@@ -137,7 +137,11 @@ bool PlaybackCursor::visible() const
 
 void PlaybackCursor::setVisible(bool arg)
 {
-    m_visible = arg;
```

Comment:
The previous implementation was better :) This method should do exactly what its name says. It's better to control the visibility outside this class